js/stronghold.js

```diff
@@ -0,0 +1,747 @@
+        return this.parseOrder (response, market);
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        const request = this.extend ({
+            'venueId': this.options['venueId'],
+            'accountId': await this.getActiveAccount (),
+            'orderId': id,
+        }, params);
+        if (!request['accountId']) {
+            throw new ArgumentsRequired (this.id + " cancelOrder requires either the 'accountId' extra parameter or exchange.options['accountId'] = 'YOUR_ACCOUNT_ID'.");
+        }
+        const response = await this.privateDeleteVenuesVenueIdAccountsAccountIdOrdersOrderId (request);
+        return this.parseOrder (response);
+    }
+
+    async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = this.extend ({
+            'venueId': this.options['venueId'],
+            'accountId': await this.getActiveAccount (),
+        }, params);
+        if (!request['accountId']) {
+            throw new ArgumentsRequired (this.id + " cancelOrder requires either the 'accountId' extra parameter or exchange.options['accountId'] = 'YOUR_ACCOUNT_ID'.");
+        }
+        const response = await this.privateGetVenuesVenueIdAccountsAccountIdOrders (request);
+        return this.parseOrders (response['result'], undefined, since, limit);
+    }
+
+    parseOrder (order, market = undefined) {
+        // { id: '178596',
+        //   marketId: 'XLMUSD',
+        //   side: 'buy',
+        //   size: '1.0000000',
+        //   sizeFilled: '0',
+        //   price: '0.10000000',
+        //   placedAt: '2019-02-01T19:47:52Z' }
+        let symbol = undefined;
+        if (market !== undefined) {
+            symbol = market['symbol'];
+        }
+        const id = this.safeString (order, 'id');
+        const datetime = this.safeString (order, 'placedAt');
+        const amount = this.safeFloat (order, 'size');
+        const price = this.safeFloat (order, 'price');
+        const filled = this.safeFloat (order, 'sizeFilled');
+        let cost = undefined;
+        let remaining = undefined;
+        if (amount !== undefined) {
+            if (filled !== undefined) {
+                remaining = amount - filled;
+            }
+            if (price !== undefined) {
+                cost = amount * price;
+            }
+        }
+        return {
+            'id': id,
+            'info': order,
+            'symbol': symbol,
+            'datetime': datetime,
+            'timestamp': this.parse8601 (datetime),
+            'side': this.safeString (order, 'side'),
+            'amount': amount,
+            'filled': filled,
+            'remaining': remaining,
+            'price': price,
+            'cost': cost,
+            'trades': [],
+            'lastTradeTimestamp': undefined,
+            'status': undefined,
```

Comment:
In this case, because the API doesn't provide an order status, I think the status should depend on the filled <= amount condition. So, if filled < amount, status = '`open`', else if filled === amount, then `'closed'`. Do you have a closed order there? Is it reported from the `{{root}}/v1/venues/{{venueId}}/accounts/:accountId/orders` endpoint? If it is, how does it look like?

Reply:
I don't think this would work because filled orders are moved to trades, and the api does not return them